src/components/Atoms/Avatar/Avatar.jsx

```diff
@@ -0,0 +1,33 @@
+import css from './Avatar.module.css';
+
+import { ReactComponent as PhotoPlaceholder } from '../../../images/icons/avatar-icon.svg';
+
+const Avatar = ({ userName, photoUrl, Header }) => {
+  if (photoUrl && Header) {
+    return (
+      <div className={css.headerPhoto}>
+        <img src={photoUrl} alt={userName} />
+      </div>
+    );
+  } else if (Header) {
+    return (
+      <div className={css.headerPlaceholder}>
+        <PhotoPlaceholder />
+      </div>
+    );
+  } else if (photoUrl) {
+    return (
+      <div className={css.headerPhoto}>
+        <img src={photoUrl} alt={userName} />
+      </div>
+    );
+  } else {
```

Comment:
Same as second condition ❌

Reply:
Actually, that was not my code. I cleared it and it seems working well now. A problem with avatar photo/placeholder was resolved as well.